skipruntime-ts/Makefile

```
@@ -1,10 +1,27 @@
 SHELL := /bin/bash
 
 SKARGO_PROFILE?=release
+NPM_WORKSPACES=\
+    -w eslint-config\
+    -w tsconfig\
+    -w skiplang/prelude/ts/binding\
+    -w skiplang/prelude/ts/wasm\
+    -w skiplang/skdate/ts\
+    -w skiplang/skjson/ts/binding\
+    -w skiplang/skjson/ts/wasm\
+    -w skipruntime-ts/api\
+    -w skipruntime-ts/core\
+    -w skipruntime-ts/helpers\
+    -w skipruntime-ts/tests\
+    -w skipruntime-ts/wasm\
+    -w skipruntime-ts/addon\
+    -w skipruntime-ts/runtime\
+    -w skipruntime-ts/server\
+    -w skipruntime-ts/examples
```

I'm not sure about quoting the shell command in the makefile, but this jq command looks like what you want:
```suggestion
NPM_WORKSPACES=$(jq --raw-output "[.workspaces[] | select(startswith(\"sql\") | not)] | map(\"-w \" + .) | .[]" package.json)
```

---

Actually, `$(shell jq` not `$(jq`, and `../package.json`:
```
NPM_WORKSPACES=$(shell jq --raw-output "[.workspaces[] | select(startswith(\"sql\") | not)] | map(\"-w \" + .) | .[]" ../package.json)
```